### PR TITLE
feat(css): add new height / width utility classes

### PR DESCRIPTION
**Pull request by johnleider**
## Description
Adds the following sizing classes:

| Class        | Description     |
| ------------ | --------------- |
| **h-auto**   | `height: auto`  |
| **h-screen** | `height: 100vh` |
| **h-0**      | `height: 0`     |
| **h-25**     | `height: 25%`   |
| **h-50**     | `height: 50%`   |
| **h-75**     | `height: 75%`   |
| **h-100**    | `height: 100%`  |

| Class        | Description     |
| ------------ | --------------- |
| **w-auto**   | `width: auto`   |
| **w-0**      | `width: 0`      |
| **w-25**     | `width: 25%`    |
| **h-33**     | `height: 33%`   |
| **w-50**     | `width: 50%`    |
| **h-66**     | `height: 66%`   |
| **w-75**     | `width: 75%`    |
| **w-100**    | `width: 100%`   |

## Motivation and Context
Add new commonly needed utility classes.

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
<template>
  <v-app>
    <v-container>
      <!-- -->
      <v-card
        v-for="size in sizes"
        :key="size"
        :class="`w-${size}`"
        class="my-3 pa-3"
        color="primary"
      >
        Width: {{ size }}%
      </v-card>
    </v-container>

    <v-container style="height: 300px">
      <v-card
        v-for="size in sizes"
        :key="size"
        :class="`h-${size}`"
        class="mx-3 pa-3"
        color="secondary"
      >
        Height: {{ size }}%
      </v-card>
    </v-container>
  </v-app>
</template>

<script>
  export default {
    name: 'Playground',
    setup () {
      return {
        sizes: ['25', '50', '75', '100'],
      }
    },
  }
</script>

```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Review by ElijahKotyluk — APPROVED** [review]

**Comment by johnleider** [comment]
Requesting **Final** comments from @vuetifyjs/core-team

File: packages/docs/src/examples/sizing/height.vue [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<template>
```
Comment (jacekkarczmarczyk) [inline packages/docs/src/examples/sizing/height.vue:1]: example seems incomplete? (same width.vue)

File: packages/vuetify/src/styles/settings/_utilities.scss [hunk 2]
```diff
@@ -478,6 +478,32 @@ $utilities: map-deep-merge(
...
       values: static relative fixed absolute sticky
+    ),
+    // Sizing
+    "height": (
+      property: height,
+      class: h,
+      values: (
+        auto: auto,
+        screen: 100vh,
+        0: 0,
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/styles/settings/_utilities.scss:489]: wouldn't it be better if it matches the grid settings? so 0, 1/12, 2/12, etc 
probably 1/12 would be used frequently but 1/3 is definitely a common size

File: packages/docs/src/examples/sizing/height.vue (hunk 1 above)
Comment (johnleider) [reply packages/docs/src/examples/sizing/height.vue:1]: I'm not doing examples until the docs are in v3.

File: packages/vuetify/src/styles/settings/_utilities.scss (hunk 2 above)
Comment (johnleider) [reply packages/vuetify/src/styles/settings/_utilities.scss:489]: what would the classes look like?

Comment (jacekkarczmarczyk) [reply packages/vuetify/src/styles/settings/_utilities.scss:489]: Hmm maybe just 2 more - `w-33` and `w-66`?
Or actually i just looked at boostrap, they have only 25s, but they have a variable for this so if user could define other sizes with a variable i default 25s are ok. 
Btw they have also `mw-100` and `mh-100` classes

**Pull request merged**